tests/test_compatibility_file.py

```
@@ -460,3 +463,52 @@ def test_calc_minimize_pressure_3d(self):
+            "pair_coeff 2 2 0.003500 0.080000 2.344000 20.985600 0.000000\n",
+            "pair_coeff 2 3 0.007700 0.178000 2.993500 42.255600 0.000000\n",
+            "pair_coeff 2 4 0.002700 0.063000 1.892400 22.990700 0.000000\n",
+            "pair_coeff 3 3 0.012000 0.263000 3.643000 85.084000 0.000000\n",
+            "pair_coeff 3 4 0.007000 0.156000 2.541900 46.293000 0.000000\n",
+            "pair_coeff 4 4 0.001200 0.046000 1.440800 25.187300 0.000000\n",
+            "\npair_modify shift yes\n",
+        ]
+        potential_lst, potential_replace = _get_potential(
+            potential=pandas.DataFrame({"Config": [potential]})
+        )
+        for i, l in enumerate(potential):
```

Ruff (0.14.6)

Ambiguous variable name: `l`

(E741)